web_src/js/features/aria-base.js

```diff
@@ -0,0 +1,7 @@
+export const ariaPatchKey = '_giteaAriaPatch';
```

Comment:
the "features" folder is already kind of unwieldly with so many files. Maybe move the aria stuff to a "aria" folder?

Reply:
I just do not want to introduce a "new" folder for this only.

I guess it could be in `js/features/aria`, it could be considered as a feature: aria feature which Fomantic UI lacks.

Reply:
Or just have everything in one file, a few hundret lines in one file don't hurt.

Reply:
On the other hand, I see benefit in having per-element files for easier removal later.

Reply:
OK, I can move the `ariaPatchKey` to each file, please help me to answer "why the key is duplicate" if there are questions from other reviewers  😁

Reply:
Oh, I guess I misunderstood some point. 

https://github.com/go-gitea/gitea/pull/23542/commits/c62f03b07713aaae627043725db9d3dfea8d0038 , the files are moved to `modules/aria` now.